Logger/Logger/Result.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logger
+{
+    internal class Result
+    {
+        public bool Status { get; set; }
+#pragma warning disable SA1201 // Elements should appear in the correct order
```

Comment:
Even if we made a decision to disable such warnings, lets stick with the GlobalSupressions approach